docs/_future/manifest.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Manifest
+lang: en
+slug: manifest
+order: 2
+layout: future
+---
+
+
+<div class="section-content">
+
+Your project should contain a `manifest.json` file that defines your app's manifest, where you can configure your app. Refer to <a href="https://api.slack.com/reference/manifests" target="_blank">app manifest documentation</a> to learn about the available manifest configurations.
+
+Notably this file informs slack of the definitions for:
+
+* [functions](/bolt-python/future/concepts#manifest-functions)
+* [workflows](/bolt-python/future/concepts#manifest-workflows)
+
+`manifest.json` should be located at the top level of your project, along with a `triggers` folder containing `*_trigger.json` files that define the <a href="https://api.slack.com/future/triggers" target="_blank">triggers</a> for your app.
+
+<pre class="structure">
+.
+├── ...
+├── manifest.json             # app manifest definition
+├── triggers                  # folder with trigger files
+│   ├── sample_trigger.json   # trigger definition
+│   └── ...
+└── ...
+</pre>
+
+---
+
+Be sure to include the following line in your `manifest.json` file in order to get type prediction & validation in your IDE.
```

Comment:
Lets put this under it's own subheader under manifest?

Reply:
Great idea I will add a sub-header for this 🥇